proto/server/PI/proto/pi_server.h

```diff
@@ -33,6 +36,9 @@ void PIGrpcServerRunAddr(const char *server_address);
 // Get port number bound to the server
 int PIGrpcServerGetPort();
 
+// Get number of PacketIn packets sent to client
+uint64_t PIGrpcServerGetPacketInCount(DeviceMgr::device_id_t device_id);
```

Comment:
Why not kill 2 birds with 1 stone and support packet-out counters with the same function?
Also, the current behavior is to "create" a device if the device_id does not exist yet and return a count of zero. Is this the desired behavior or should we return some kind of error?

Reply:
I added new API PIGrpcServerGetPacketOutCount. I thought that would be neater. Fixed current behavior to not create DeviceState if a device_id does not exist. I think it is fine to return 0 in case device_id does not exist yet.  Technically, we didn't receive/send any packets from/to non-existent device.